src/main/java/iroha/validation/service/impl/ValidationServiceImpl.java

```diff
@@ -1,5 +1,39 @@
 package iroha.validation.service.impl;
 
-public class ValidationServiceImpl {
+import iroha.validation.config.ValidationServiceContext;
+import iroha.validation.service.ValidationService;
+import iroha.validation.transactions.signatory.TransactionSigner;
+import iroha.validation.transactions.storage.TransactionProvider;
+import iroha.validation.validators.Validator;
+import java.security.KeyPair;
+import java.util.Collection;
 
+public class ValidationServiceImpl implements ValidationService {
+
+  private Collection<Validator> validators;
+  private TransactionProvider transactionProvider;
+  private TransactionSigner transactionSigner;
+  private KeyPair keyPair;
+
+  public ValidationServiceImpl(ValidationServiceContext validationServiceContext) {
```

Comment:
Can validationServiceContext be null? Looks like not

Reply:
A check has been added